src/lib/processor/process_get_open_orders.rs

```diff
@@ -54,6 +80,24 @@ pub async fn process_get_open_orders(
                     order.num_base_lots.as_u64() as f64 * sdk.base_lots_to_base_units_multiplier(),
                     size_precision
                 ),
+                format!(
+                    "{}",
+                    if order.last_valid_slot >= clock.slot {
+                        (1 + order.last_valid_slot - clock.slot).to_string()
+                    } else {
+                        "∞".to_string()
```

Comment:
does this need to be inside its own `format!`? can you just pass the raw string value to the line 70 `format!`?